### DISCUSSION TITLE
Добавить предыдущее

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#522] [ОШБК!]
Скажите пожалуйста, это такой баг или я что-то не так настроил? Не могу разобраться.

[Проекты — ERGO.webm](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/71fd8121-3f0d-45d2-bccd-ace557d10aaa)

**Comment by ttm-alexeypolunin** [comment]
По видимым на экране признакам у нас не повторяется
